common/src/main/java/io/netty/util/DefaultAttributeMap.java

```diff
@@ -22,10 +22,11 @@
 import java.util.concurrent.atomic.AtomicReferenceFieldUpdater;
 
 /**
- * Default {@link AttributeMap} implementation which use simple synchronization per bucket to keep the memory overhead
+ * Default {@link DefaultAttributeMap} implementation which
+ * use simple synchronization per bucket to keep the memory overhead
  * as low as possible.
  */
-public class DefaultAttributeMap implements AttributeMap {
+public class DefaultAttributeMap {
```

Comment:
@doom369 should we merge this class with `AbstractChannel` ?

Reply:
@normanmaurer hm... Actually, very good idea. Didn't think about it. Let me check.